hw/ip/aes/data/aes.hjson

```diff
@@ -194,11 +209,81 @@
       '''
     }
   ],
+
+  /////////////////////
+  // Countermeasures //
+  /////////////////////
+
   countermeasures: [
     { name: "BUS.INTEGRITY",
       desc: "End-to-end bus integrity scheme."
     }
+    { name: "CONFIG.SHADOW",
+      desc: "Main control register shadowed."
+    }
+    { name: "CONFIG.SPARSE",
+      desc: "Critical fields in main control register one-hot encoded."
+    }
+    { name: "DATA_REG.SW_UNREADABLE",
+      desc: "Key, IV and data input registers are write only for software."
+    }
+    { name: "DATA_REG.SEC_WIPE",
+      desc: "Key, IV, input/output data and internal state registers can be cleared with pseudo-random data."
```

Comment:
Are there situations where they're cleared with something else? Otherwise, "are cleared" might be better?